### PR TITLE
adding Caffe 1.0 and dependencies

### PR DESCRIPTION
**Pull request by gppezzi**
Sanity check command fails during build, but it works after installing (that's odd).

### CODE REVIEW

File: easybuild/easyconfigs/c/Caffe/Caffe-1.0-CrayGNU-2016.11-cuda-8.0-Python-2.7.12.eb [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# @author: gppezzi
+
+easyblock = 'CMakeMake'
+
+name = 'Caffe'
+version = '1.0'
+
+cudaversion = '8.0'
+versionsuffix = '-cuda-%s-Python-%%(pyver)s' % cudaversion
```
Comment (jgphpc) [inline easybuild/easyconfigs/c/Caffe/Caffe-1.0-CrayGNU-2016.11-cuda-8.0-Python-2.7.12.eb]: need `pyver = '2.7.12'` ?

File: easybuild/easyconfigs/c/Caffe/Caffe-1.0-CrayGNU-2016.11-cuda-8.0-Python-2.7.12.eb [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    ('Boost', '1.63.0', '-Python-%(pyver)s'),
+    ('Python', '2.7.12'),
+    ('scikit-image', '0.13.0', '-Python-%(pyver)s')
+]
+
+buildininstalldir = True
+
+configopts = '-DBLAS=libsci '
+configopts += '-DBOOST_ROOT=$EBROOTBOOST '
+configopts += '-DProtobuf_INCLUDE_DIR=$EBROOTPROTOBUFMINCORE/include  -DProtobuf_LIBRARY=$EBROOTPROTOBUFMINCORE/lib/libprotobuf.so '
```
Comment (jgphpc) [inline easybuild/easyconfigs/c/Caffe/Caffe-1.0-CrayGNU-2016.11-cuda-8.0-Python-2.7.12.eb]: split into 2 lines for easier reading ?

File: easybuild/easyconfigs/g/glog/glog-0.3.4-CrayGNU-2016.11.eb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+easyblock = 'ConfigureMake'
+
+name = 'glog'
+version = '0.3.4'
+
+homepage = 'https://github.com/google/glog'
+description = """
+A C++ implementation of the Google logging module.
```
Comment (jgphpc) [inline easybuild/easyconfigs/g/glog/glog-0.3.4-CrayGNU-2016.11.eb]: why \n ?

File: easybuild/easyconfigs/l/LMDB/LMDB-0.9.18-CrayGNU-2016.11.eb [hunk 4]
```diff
@@ -0,0 +1,26 @@
+easyblock = 'MakeCp'
+
+name = 'LMDB'
+version = '0.9.18'
+
+homepage = 'https://github.com/LMDB/lmdb'
+description = """
+OpenLDAP's Lightning Memory-Mapped Database (LMDB) library.
```
Comment (jgphpc) [inline easybuild/easyconfigs/l/LMDB/LMDB-0.9.18-CrayGNU-2016.11.eb]: why \n ?

File: easybuild/easyconfigs/p/Pillow/Pillow-3.2.0-CrayGNU-2016.11-Python-2.7.12-freetype-2.6.3.eb [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+homepage = 'http://pillow.readthedocs.org/'
+description = """Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.
+ PIL is the Python Imaging Library by Fredrik Lundh and Contributors."""
+
+toolchain = {'name': 'CrayGNU', 'version': '2016.11'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+freetype_ver = '2.6.3'
```
Comment (jgphpc) [inline easybuild/easyconfigs/p/Pillow/Pillow-3.2.0-CrayGNU-2016.11-Python-2.7.12-freetype-2.6.3.eb]: move up to after line 5 ?

File: easybuild/easyconfigs/p/Pillow/Pillow-3.2.0-CrayGNU-2016.11-Python-2.7.12-freetype-2.6.3.eb [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+versionsuffix += '-freetype-%s' % freetype_ver
+
+dependencies = [
+    ('Python', '2.7.12'),
```
Comment (jgphpc) [inline easybuild/easyconfigs/p/Pillow/Pillow-3.2.0-CrayGNU-2016.11-Python-2.7.12-freetype-2.6.3.eb]: pyver

File: easybuild/easyconfigs/c/Caffe/Caffe-1.0-CrayGNU-2016.11-cuda-8.0-Python-2.7.12.eb (hunk 1 above)
Comment (gppezzi) [inline easybuild/easyconfigs/c/Caffe/Caffe-1.0-CrayGNU-2016.11-cuda-8.0-Python-2.7.12.eb]: pyver is automatically defined with
```
    ('Python', '2.7.12'),
```

**Pull request merged**